eng/targetframeworksuffix.props

```diff
@@ -88,4 +87,8 @@
       </PropertyGroup>
     </When>
   </Choose>
+
+  <PropertyGroup>
+    <EnablePInvokeAnalyzer Condition="'$(TargetsBrowser)' == 'true'">false</EnablePInvokeAnalyzer>
```

Comment:
why do we need to disable the pinvoke analyzer for browser?

Reply:
Pinvoke analyzer works just for window. By default we disable it for unix.

Reply:
where is that defined for unix? why aren't we doing that in the same place?

Reply:
PInvokeAnalyzer was disabled accross the board in https://github.com/dotnet/runtime/pull/38550 . This is adding dead code.